src/Library.java

```diff
@@ -1,3 +1,4 @@
+//File ready for static review - Michael Coleman 11627449
```

Comment:
line 21 - 26 final variables need to be renamed according to constant names. google java style guide 5.2.4.